.github/workflows/dev.yml

```diff
@@ -170,9 +203,33 @@ jobs:
           just \
             debug_justfile="${{matrix.debug_justfile}}" \
             profile=${{matrix.profile.name}} \
-            registry="${{env.UPSTREAM_REGISTRY}}" \
+            dpdp_sys_registry="${{env.UPSTREAM_REGISTRY}}" \
             target=x86_64-unknown-linux-gnu \
-            push-container
+            version=${{ needs.version.outputs.version }}-${{ matrix.profile.name }} \
+            oci_repo="ghcr.io" \
+            push
+
+      - name: Comment on PR with temp artifacts version
+        if: ${{ github.event_name == 'pull_request' && matrix.profile.sterile == 'sterile' && matrix.profile.name == 'release' }}
+        uses: actions/github-script@v8
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: '🚀 Temp artifacts published: `${{ steps.version-gen.outputs.version }}` 🚀'
+            })
```

Comment:
I can't speak for others, but I really don't like these comments. I find they make too much noise both on the PR page, and in my mail inbox; and I guess we'll get a new one each time we force-push.

How often do you use these links? Would it be enough to print them in the workflow logs, rather than a PR comment?